File: WinForm Demo (C#)/MySqlBackupDemo/frmInsertBlob.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using MySql.Data.MySqlClient;

namespace MySqlBackupDemo
{
    public partial class frmInsertBlob : Form
    {
        string constr = "";
        public frmInsertBlob(string connectionString)
        {
            constr = connectionString;
            InitializeComponent();

            MySqlBackup mb = new MySqlBackup(connectionString);
            foreach (string s in mb.DatabaseInfo.TableNames)
            {
                cbTable.Items.Add(s);
            }
            cbTable.SelectedIndex = -1;
            this.cbTable.SelectedIndexChanged += new EventHandler(cbTable_SelectedIndexChanged);
        }

        void cbTable_SelectedIndexChanged(object sender, EventArgs e)
        {
            bool hasBLOB = false;
            using (MySqlConnection conn = new MySqlConnection(constr))
            {
                conn.Open();
                string sql = "show columns from `" + cbTable.Text + "`;";
                DataTable dt = new DataTable();
                MySqlDataAdapter da = new MySqlDataAdapter(sql, conn);
                da.Fill(dt);
                cbColBLOB.Items.Clear();
                foreach (DataRow dr in dt.Rows)
                {
                    string s = dr["Type"] + "";
                    if (s.ToLower().Contains("blob"))
                    {
                        hasBLOB = true;
                        cbColBLOB.Items.Add(dr["Field"] + "");
                    }
                    else
                    {
                        cbColFileName.Items.Add(dr["Field"] + "");
                        cbColFileSize.Items.Add(dr["Field"] + "");
                      
                    }
                }

                try
                {
                    cbColBLOB.SelectedIndex = 0;
                    cbColFileSize.SelectedIndex = 1;
                    cbColFileName.SelectedIndex = 0;
                }
                catch { }
                conn.Close();
            }

            if (!hasBLOB)
            {
                MessageBox.Show("This table has no BLOB data type column.");
            }
        }

        private void frmInsertBlob_Load(object sender, EventArgs e)
        {

        }

        private void btFile_Click(object sender, EventArgs e)
        {
            OpenFileDialog of = new OpenFileDialog();
            of.Multiselect = false;
            if (of.ShowDialog() != System.Windows.Forms.DialogResult.OK)
                return;
            txtFile.Text = of.FileName;
        }

        private void btInsert_Click(object sender, EventArgs e)
        {
            if (!System.IO.File.Exists(txtFile.Text))
            {
                MessageBox.Show("The specify file does not exists.");
                return;
            }
            using (MySqlConnection conn = new MySqlConnection(constr))
            {
                MySqlCommand cmd = new MySqlCommand();
                cmd.Connection = conn;
                conn.Open();
                Methods md = new  Methods();
                byte[] ba = System.IO.File.ReadAllBytes(txtFile.Text);
                

                for (int i = 0; i < (int)numericUpDown1.Value; i++)
                {
                    string sql = "insert into `" + cbTable.Text + "`(`" + cbColBLOB.Text + "`, `" + cbColFileName.Text + "`,`" + cbColFileSize.Text + "`) values(" + md.GetBLOBSqlDataStringFromBytes(ba) + ",'" + System.IO.Path.GetFileNameWithoutExtension(txtFile.Text) + (i + 1) + System.IO.Path.GetExtension(txtFile.Text) + "'," + ba.Length + ");";
                    cmd.CommandText = sql;
                    cmd.ExecuteNonQuery();
                }

                md = null;
                cmd = null;

                conn.Close();
            }
            MessageBox.Show("Done.");
        }
    }
}

File: WinForm Demo (VB.NET)/MySqlBackupDemoVB/frmInsertBlob.vb
Imports MySql.Data.MySqlClient
Public Class frmInsertBlob

    Private constr As String = ""

    Public Sub New(ByVal connectionString As String)

        constr = connectionString
        InitializeComponent()

        Dim mb As New MySqlBackup(connectionString)

        For Each s As String In mb.DatabaseInfo.TableNames
            cbTable.Items.Add(s)
        Next

        cbTable.SelectedIndex = -1
        AddHandler Me.cbTable.SelectedIndexChanged, AddressOf cbTable_SelectedIndexChanged

    End Sub


    Private Sub frmInsertBlob_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load

    End Sub


    Private Sub btFile_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btFile.Click
        Using ofd As New OpenFileDialog()
            ofd.Multiselect = False
            If ofd.ShowDialog() <> System.Windows.Forms.DialogResult.OK Then Exit Sub
            txtFile.Text = ofd.FileName
        End Using
    End Sub

    Private Sub btInsert_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btInsert.Click

        If String.IsNullOrEmpty(txtFile.Text.Trim) Then
            MessageBox.Show("File is not specified.")
            Exit Sub
        ElseIf Not System.IO.File.Exists(txtFile.Text) Then
            MessageBox.Show("The specified file does not exists.")
            Exit Sub
        End If

        Dim ErrorMsg As String = ""

        Using conn As New MySqlConnection(constr)

            conn.Open()

            Dim cmd As New MySqlCommand()
            cmd.Connection = conn

            Dim md As New Methods()
            Dim ba As Byte() = System.IO.File.ReadAllBytes(txtFile.Text)

            Try
                For i As Integer = 0 To CInt(numericUpDown1.Value) - 1
                    Dim sql As String = "insert into `" & cbTable.Text & "`(`" _
                        & cbColBLOB.Text & "`, `" & cbColFileName.Text & "`,`" _
                        & cbColFileSize.Text & "`) values(" & md.GetBLOBSqlDataStringFromBytes(ba) _
                        & ",'" & System.IO.Path.GetFileNameWithoutExtension(txtFile.Text) _
                        & (i + 1) & System.IO.Path.GetExtension(txtFile.Text) & "'," & ba.Length & ");"
                    cmd.CommandText = sql
                    cmd.ExecuteNonQuery()
                Next
            Catch ex As Exception
                ErrorMsg = ex.Message
            End Try

            If conn.State = ConnectionState.Open Then conn.Close()
            conn.Dispose()
            cmd.Dispose()
            md = Nothing

        End Using

        If String.IsNullOrEmpty(ErrorMsg.Trim) Then
            MessageBox.Show("Done.")
        Else
            MessageBox.Show("Error occured while inserting: " & ErrorMsg.Trim)
        End If

    End Sub

    Private Sub cbTable_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs)

        Dim hasBLOB As Boolean = False

        Using conn As New MySqlConnection(constr)

            conn.Open()

            Dim sql As String = "show columns from `" + cbTable.Text + "`;"
            Dim dt As New DataTable()
            Dim da As New MySqlDataAdapter(sql, conn)
            da.Fill(dt)

            conn.Close()

            cbColBLOB.Items.Clear()
            For Each dr As DataRow In dt.Rows
                Dim s As String = dr("Type") & ""
                If s.ToLower().Contains("blob") Then
                    hasBLOB = True
                    cbColBLOB.Items.Add(dr("Field") & "")
                Else
                    cbColFileName.Items.Add(dr("Field") & "")

                    cbColFileSize.Items.Add(dr("Field") & "")
                End If
            Next

            dt.Dispose()

            Try
                cbColBLOB.SelectedIndex = 0
                cbColFileSize.SelectedIndex = 1
                cbColFileName.SelectedIndex = 0
            Catch
            End Try

        End Using

        If Not hasBLOB Then MessageBox.Show("This table has no BLOB data type column.")

    End Sub

End Class